templates/highlighter.css

```diff
@@ -0,0 +1,12 @@
+.__popcode-highlighter {
+  position:absolute;
+  z-index: 10000000000;
+  border:1px solid rgba(238, 238, 48, 0.2);
+  box-sizing:border-box;
+  box-shadow:0 0 20px rgba(0,0,0,0.2);
+  pointer-events: none;
+  color: black;
+  background: transparent;
+  box-sizing: border-box;
```

Comment:
Hmm, won’t this mean the border itself sits on top of the inner edge of the highlighted element? Could we instead use `content-box` and just give it a -1px margin top/left?